spec/controllers/reviews_controller_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'spec_helper'
+
+describe ReviewsController do
+
+  describe 'POST :create' do
+
+    let(:video) { video = Fabricate(:video) }
+
+    context 'user is authorized' do
+      let(:user) { user = Fabricate(:user) }
+      before do
+        self.controller.login_user user
```

Comment:
Rather than this I would prefer building a RSpec macro "sign_in user". Asking controller to log in the user feels ..weird to me.